saleor/payment/gateways/stripe/plugin.py

```diff
@@ -430,6 +430,7 @@ def list_payment_sources(
                     ),
                 )
                 for c in payment_methods
+                if c.metadata.get("channel", channel_slug) == channel_slug
```

Comment:
so unless there is `channel` in metadata, we assume it's the same as `channel_slug`? Is the 2nd `.get` parameter intentional?
Besides - what does the var name `c` stands for? It's payment method, so maybe `p` or `payment` would be more accurate?